src/main/scala/tech/cryptonomic/conseil/tezos/michelson/renderer/MichelsonRenderer.scala

```diff
@@ -52,7 +52,7 @@ object MichelsonRenderer {
       self
         .map(_.render())
         .mkString(" ;\n")
-        .lines
+        .linesIterator
```

Comment:
pull master into this branch, this was fixed by @piotrkosecki in a different pr

Reply:
I will, sure, I fixed it here before